rpm_spec/core-dom0-linux.spec.in

```diff
@@ -50,6 +50,8 @@ Conflicts:	qubes-core-dom0 < 4.0.23
 Requires:	%{name}-kernel-install
 Requires:	xdotool
 Requires:	createrepo_c
+Requires:	qubes-core-dom0-vaio-fixes
+Requires:	qubes-core-dom0-macbook12-spi-driver
```

Comment:
The idea of those packages being separate is to _not_ install them by default. Specifically, vaio-fixes (which is probably horribly outdated now) do some changes that may be harmful on other systems. So, it is wrong to add it as a dependency here.
If the same applies to your new package - please do _not_ add it here. On the first sight, I don't think this would be break anything, but will make initramfs slightly bigger. If you want it by default, please add a code including those drivers only on relevant hardware. You can do that with dracut module (instead of just config) - for that create `/usr/lib/dracut/modules.d/90macbook12-spi-driver/module-setup.sh` (or similar). [Here you have documentation](http://man7.org/linux/man-pages/man7/dracut.modules.7.html) and [here an example](https://github.com/QubesOS/qubes-linux-utils/blob/master/dracut/full-dmroot/module-setup.sh)

Also, since creation of the vaio-fixes package, many things have changed in preferred packaging. Specifically, we prefer a subpackage defined in the same spec file, instead of separate spec file. See the main spec and how `kernel-install` subpackage is done.

Reply:
Much appreciate for the pointers. Will do as soon as I get some time.